internal/services/signalr/web_pubsub_hub_resource.go

```diff
@@ -95,12 +94,9 @@ func resourceWebPubSubHub() *pluginsdk.Resource {
 							Elem: &pluginsdk.Resource{
 								Schema: map[string]*pluginsdk.Schema{
 									"managed_identity_id": {
-										Type:     pluginsdk.TypeString,
-										Required: true,
-										ValidateFunc: validation.Any(
-											validation.IsUUID,
-											commonids.ValidateUserAssignedIdentityID,
-										),
+										Type:         pluginsdk.TypeString,
+										Required:     true,
+										ValidateFunc: validation.StringIsNotEmpty,
```

Comment:
rather than removing the existing validation, would it be possible to include additional validation for the token audience instead?

Reply:
@catriona-m Thanks for the review! I checked the api definition and also confirmed with the service team. There isn't any validation to the user's input of the string, any string set to this `managed_identity_id` will be accepted by the api, however, there will be error when the token is actually used. So I don't think we need to add any validation to it neither, WDYT?

Reply:
thanks @xiaxyi - if this can be any string then I think we can leave this as it is